templates/etc/dhcp/dhcpd.conf.j2

```diff
@@ -45,13 +45,13 @@ failover peer "failover" {
 failover peer "failover" {
   secondary;
   address {{ isc_dhcp_failover_info.failover_address }};
-  port 520;
+  port {{ isc_dhcp_failover_info.secondary_port|default('520') }};
 {%     for host in groups[isc_dhcp_failover_info.failover_group] %}
 {%       if inventory_hostname != host %}
   peer address {{ host }};
 {%       endif %}
 {%     endfor %}
-  peer port 519;
+  peer port {{ isc_dhcp_failover_info.secondary_port|default('519') }};
```

Comment:
Shouldn't this be `isc_dhcp_failover_info.primary_port|default('519')`

Reply:
Indeed, fixed. Thanks for spotting this!